docs/specification.md

```diff
@@ -363,10 +360,9 @@ custom_permission_policy =
   )
 
 permissions_descriptor_param
-  ( self_transfer_policy            :self
-  , operator_transfer_policy        :receiver
-  , owner_transfer_policty          :operator
-  , owner_transfer_policty          :sender
+  ( operator_transfer_policy        :receiver
```

Comment:
The annotations seems to be mixed up here.`operator_transfer_policy` needs `:operator` annotation, right?

Reply:
yep